### ISSUE TITLE
Open files using emulator?

### ISSUE THREAD

**Issue by lhoezee** [markeeftb/FileOpener#4]
Hello, I'm not sure if this is an "issue", more of a question.  Should I be able to open files with this plugin with an emulator?  Currently, I'm not getting any errors, however nothing happens after calling : 

```
window.plugins.fileOpener.open(filePath);
```

Also, I know I'm getting into here when I alert as well:

```
FileOpener.prototype.open = function(url) {
    cordova.exec(null, null, "FileOpener", "openFile", [url]);
};
```

**Comment by Anj1ana** [comment]
Have you got the solution?? I am also having same problem. My PDF is not opening. I m also using emulator.

**Comment by lhoezee** [comment]
I don't believe so.  I think I used another plugin or created my own.  Honestly I can not even remember what project I was using this on.  I attempted to grep all my projects for "fileOpener" and I'm not finding anything which tells me I tried a completely different solution.  Sorry.

**Comment by Anj1ana** [comment]
Okay.. np

**Comment by JaviDTM** [comment]
How I can open a PDF that is in my www folder?
